tools/wake-format/types.h

```diff
@@ -47,6 +47,7 @@ enum class ExplodeOption {
 
 struct ctx_t {
   size_t nest_level = 0;
+  size_t prefix_length = 0;
```

Comment:
This needs a comment and a better name. "prefix" is too generic for how specific this appears to be. As is I have no idea what this is for other than some highly specific thing for fixing this issue that I don't yet understand.

Reply:
fair